SpatialGDK/Source/SpatialGDK/Public/SpatialGDKSettings.h

```diff
@@ -70,29 +70,55 @@ class SPATIALGDK_API UEventTracingSamplingSettings : public UObject
 {
 	GENERATED_BODY()
 public:
+	struct TraceQueryDeleter
```

Comment:
Why move this?

Reply:
This was to allow the UEventTracingSamplingSettings to check the filter strings itself. UEventTracingSamplingSettings is now able to use the worker APIs to validate filter strings. This allows UEventTracingSamplingSettings to always return valid filters. We need filters in two areas of the code so it makes sense to do the checks at the root.